ui/components/app/clear-metametrics-data/clear-metametrics-data.test.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import configureStore from '../../../store/store';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+
+import ClearMetaMetricsData from './clear-metametrics-data';
+
+describe('ClearMetaMetricsData', () => {
```

Comment:
Can we also add a unit test that checks if there is a button and that `createMetaMetricsDataDeletionTask` is called if the button is clicked?